src/new-block/elements/get-link.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import clsx from 'clsx';
+
+/**
+ * Internal dependencies
+ */
+import type { Attrs } from '../blocks/link/edit';
+
+export type Props = Attrs;
+
+export default function getLink( { linkUrl, text, className }: Props ): string {
+	return `
```

Comment:
Just to confirm, you are returning a string instead of a component intentionally?

Reply:
Yes, `/elements/*` are being used by the [frontend view](https://github.com/Automattic/gravatar-enhanced/pull/9/files#diff-945f59553bd876d25e468f80ee7cf536df1f987cbf580dd5c1d89782a6a95ce6R200). We will reuse these elements to create patterns ( p8Slzc-3SR-p2#patterns ).